docs/documentation/upgrading/topics/keycloak/changes-22_0_0.adoc

```diff
@@ -74,3 +77,56 @@ NOTE:: When the auto-build runs, the build time options will be in effect for al
 In previous releases the `export` and `import` commands allowed runtime parameters like, for example, a database URL only in configuration files or environment variables.
 Starting with this release, those runtime parameters are now available on the command line as well.
 Use the `--help` option to find out about the supported parameters.
+
+= Renamed Keycloak Admin client dependencies
```

Comment:
Actually, one more. Sorry, missed it before.
```suggestion
= Renamed Keycloak Admin client artifacts
```

Reply:
Oh, me too. Thanks @vmuzikar!